features/admin/edit_fein.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Update FEIN
```

Comment:
Please make all these three scenarios into single scenario outline

Reply:
updated scenarios into single scenario outline. Thanks for suggestion.